examples/fluent-bit/cloudwatchlogs-emf/emf-over-tcp/extra.conf

```diff
@@ -24,3 +24,6 @@
     log_stream_prefix   from-fluent-bit-
     auto_create_group   true
     log_format          json/emf
+    # from aws-for-fluent-bit v2.32.0 and on, to support higher throughput logging,
+    # set workers to a high value such as 5 or the number of cores on your host
```

Comment:
I'd rephrase this so rather than being a command its like, "consider configuring more than 1 worker, up to the number of cores on your host"

and I'd also explicitly say only 2.32+ supports multi worker

Reply:
phrase it that way

Reply:
Sure! Updated.